internal/rbd/group.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2024 The Ceph-CSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rbd
+
+import (
+	"context"
+	"fmt"
+
+	librbd "github.com/ceph/go-ceph/rbd"
+
+	"github.com/ceph/ceph-csi/internal/rbd/types"
+)
+
+// AddToGroup adds the image to the group. This is called from the rbd_group
+// package.
+func (rv *rbdVolume) AddToGroup(ctx context.Context, vg types.VolumeGroup) error {
+	ioctx, err := vg.GetIOContext(ctx)
```

Comment:
ioctx is not used

Reply:
yes it is, on `return librbd.GroupImageAdd(ioctx, ...`.

Reply:
but you are passing rv.ioctx not the one you get from here

Reply:
ha my mistake, it looks like we have two ioctx